plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -338,8 +342,15 @@ class NewCreateServiceModalForm extends Component {
               onRemoveItem={this.handleRemoveItem}
               onAddItem={this.handleAddItem} />
         </TabView>,
-        <TabView id="networking" key="networking">
-          <NetworkingFormSection
+        <TabView id="volumes" key="multivolumes">
+          <VolumeMountFormSection
+              data={data}
```

Comment:
Indented one too much. I've seen this _a lot_ lately. Could you fix the editor config or your editor settings?